internal/config/migrate/from_1_17.go

```diff
@@ -61,8 +61,8 @@ func migrateFrom1_17(cfg *config.Config) error {
 
 	// Upgrade pause image
 	// https://github.com/cri-o/cri-o/pull/4550
-	logrus.Infof("Checking for pause_image, which now should be %s instead of k8s.gcr.io/pause:3.1 or 3.2", config.DefaultPauseImage)
-	if cfg.PauseImage == "k8s.gcr.io/pause:3.1" || cfg.PauseImage == "k8s.gcr.io/pause:3.2" {
+	logrus.Infof("Checking for pause_image, which now should be %s instead of registry.k8s.io/pause:3.1 or 3.2", config.DefaultPauseImage)
+	if cfg.PauseImage == "registry.k8s.io/pause:3.1" || cfg.PauseImage == "registry.k8s.io/pause:3.2" {
```

Comment:
@saschagrunert @sohankunkerkar we practically haven't supported config migration since this version. should we drop this option? can be done as a follow up

Reply:
yup, works for me